### ISSUE TITLE
Should `@testsnippet` be reexported by this package?

### ISSUE THREAD

**Issue by juliohm** [julia-vscode/TestItemRunner.jl#91]
When we add `using TestItemRunner` in the beginning of runtests.jl it exports all macros, except `@testsnippet`. We needed to also import `TestItems` to get things working.

**Comment by davidanthoff** [comment]
The idea really is that there are no test items (nor `include`s) in `runtests.jl`, just the macro call to run the tests, so that one doesn't need to load TestItems.jl. That should then work, right?

**Comment by juliohm** [comment]
Exactly. As a first time user I thought that TestItemRunner.jl was the only
package needed to write the test suite.

Em sáb., 24 de ago. de 2024, 23:32, David Anthoff ***@***.***>
escreveu:

**Comment by ianfiske** [comment]
I can confirm that it works as-is if you move your `@testsnippet` to a separate file from the `runtests.jl` file.

**Comment by juliohm** [comment]
So moving the snippet to any other file like `test/snippets.jl` removes the need of the dependency on TestItems.jl?

**Comment by davidanthoff** [comment]
The macro is now exported.
